### PR TITLE
Gemstones: add torsocks entry

### PR DESCRIPTION
**Pull request by neelchauhan**
#### Author checklist (Completed by original Author)
- [X] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [X] If applicable, steps and instructions have been tested to work
- [X] Initial self-review to fix basic typos and grammar completed

#### Rocky Documentation checklist (Completed by Rocky team) 
- [x] 1st Pass (Document is good fit for project and author checklist completed)
- [x] 2nd Pass (Technical Review - check for technical correctness) 
- [x] 3rd Pass (Detailed Editorial Review and Peer Review)
- [x] Final approval (Final Review)

### CODE REVIEW

File: docs/gemstones/torsocks.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+|-p PASS |Set the SOCKS5 password|
+|-a ADDRESS |Set the SOCKS5 server address|
+|-P PORT |Set the SOCKS5 server port|
+|-i |Enable Tor isolation)
+
+A sample (redacted) output from the IP checker [icanhazip.com](https://icanhazip.com/) via `torsocks`:
+
+![torsocks output](./images/torsocks.png)
+
+Notice that the `torsocks` IP address is different from the direct `curl` IP addres.
```
Comment (SergeCroise) [inline docs/gemstones/torsocks.md:35]:
```suggestion
Notice that the `torsocks` IP address is different from the direct `curl` IP address.
```

File: docs/gemstones/torsocks.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## Using `torsocks`
+
+```bash
+dnf -y install epel-release
+dnf -y install tor torsocks
+systemctl enable --now tor
+```
+
+The common options of the `torsocks` command follow, and under normal circumstances, require nothing additional. The options come before the application to run (e.g. `curl`):
```
Comment (SergeCroise) [inline docs/gemstones/torsocks.md:20]:
```suggestion
The common options of the `torsocks` command follow, and under normal circumstances, require nothing additional. The options come before the application to run (e.g., `curl`):
```

File: docs/gemstones/torsocks.md (hunk 1 above)
Comment (neelchauhan) [reply docs/gemstones/torsocks.md]: done!

File: docs/gemstones/torsocks.md (hunk 2 above)
Comment (neelchauhan) [reply docs/gemstones/torsocks.md]: done!

**Review by sspencerwire — APPROVED** [review]

**Pull request merged**